examples/README.md

```diff
@@ -0,0 +1,228 @@
+# Examples
+
+
+### Simple example
+
+*simple-example-blueprint.yaml*
+
+TODO
+
+### Replicasets
+
+*replicasets-example-blueprint.yaml*
+
+TODO
+
+### Persistent volumes
+
+*persistent_volumes-example-blueprint*
+
+TODO
+
+### Service chaining
+
+There are 3 blueprints defined as examples of container-based service chaining for kubernetes.
+These scenarios are using Linux bridging and static routing to provide chain connectivity between separate pods.
+Implementations of all scenarios are done using utilities-plugin.
+Separate generic blueprint *vnf-blueprint* is used to define each pod and network interfaces by *service_chain* deployment.
+In other words main (*service_chain*) deployment is creating separate deployments from *vnf-blueprint* for each pod using utilities-plugin.
+  
+So, before start you need to upload *vnf-blueprint* to Cloudify Manager. You can do it by:
+
+```
+cfy blueprints upload vnf-blueprint.yaml -b service_chain_vnf_component

```

Comment:
vnf-blueprint in only reusable part to be used with service chain blueprint. I will provide example input file for vnf-blueprint but in my opinion it shouldn't be run as separate deployment.